src/lib/routes/index.ts

```diff
@@ -47,11 +46,7 @@ class IndexRouter extends Controller {
 
         this.use(
             '/api/frontend',
-            conditionalMiddleware(
```

Comment:
conditionalMiddleware is hiding openapi definitions so I moved the flag inside the 3 controller methods in the frontend API